src/misc/portal.ts

```diff
@@ -59,6 +59,18 @@ export class Portal extends Component<any, any> {
       }
     });
   }
+  /**
+   * Override to revert back to a classic Component's structure
+   *
+   * @override
+   */
+  __callWillUnmount() {
+    super.__callWillUnmount();
+    if (this.el && this.portal && this.portal.elm) {
```

Comment:
I think that if you component is being unmounted, then it necessarily has a `el`, `portal` and `portal.elm`, so you can remove this `if`